nautobot/core/management/commands/send_installation_metrics.py

```diff
@@ -0,0 +1,67 @@
+import hashlib
+import platform
+import requests
+import uuid
+
+from constance import config
+from django.apps import apps
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from nautobot.utilities.config import get_settings_or_config
+
+
+METRICS_ENDPOINT = "https://nautobot.cloud/api/nautobot/installation-metric/"
+
+
+class Command(BaseCommand):
+    help = "Send installation metrics for this Nautobot installation."
+
+    def _hash(self, plaintext):
+        return hashlib.sha256(plaintext.encode("utf8")).hexdigest()
+
+    def get_hashed_plugins_with_version(self):
+        plugins = {}
+        for plugin_name in settings.PLUGINS:
+            plugin_name = plugin_name.rsplit(".", 1)[-1]
```

Comment:
What's this for? Handling any potential for plugins to be `foo.bar` --> just `bar`?

Reply:
I suspect this is from @gsnider2195's original implementation which used the app list instead of the settings.PLUGINS?

Reply:
This was copy pasted from the view code (`/status` I think?)

Reply:
Yeah looks the same there. I'm not sure it's necessary but it's probably not hurting anything either.